tests/test_odgovlt.py

```diff
@@ -195,3 +203,13 @@ def test_OdgovltHarvester(app, db, mocker):
     assert result['report_status'] == 'added'
     assert result['errors'] == []
     assert was_last_job_considered_error_free()
+    tag1 = database_data_list[0]['R_ZODZIAI']
+    tag2 = database_data_list[1]['R_ZODZIAI']
+    tag_1 = tag1.lower().replace(u'\u200b', '').split(',')
+    tag_2 = tag2.lower().replace(u'\u200b', '').split(',')
+    tag = tag_1 + tag_2
+    context = {'model': model, 'session': model.Session, 'ignore_auth': True}
+    tag_list = get_action('tag_list')(context, {})
+    assert tag_list
+    assert isinstance(tag_list, list)
+    assert set(tag) == set(tag_list)
```

Comment:
You test everything with `package_show` and `package_list`, like I did here:

https://github.com/ivpk/opendata.gov.lt-mysql-import/blob/3c22ae02e2b7114d1cb8e66eaed6de4d09f529fa/tests/test_import.py#L173